src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -493,6 +493,26 @@ type Todo @model(subscriptions: { level: off }) { # or level: public
 }
 ```
 
+### Create multiple relationships between two models
+
+You need to explicitly specifying the connection field names, if relational directives are used to create two connections of the same type between the two models.
+
+```graphql
+type Individual @model {
+  id: ID!
+  homeAddress: Address @hasOne
+  shippingAddress: Address @hasOne
+}
+
+type Address @model {
+  id: ID!
+  homeIndividualID: ID @index(name: "homeIndex")
+  shippingIndividualID: ID @index(name: "shipIndex")
```

Comment:
I think the use of `@index` here distracts from what the example is about. What do you think?

Reply:
Yeah that's fair, I can cut it out of the schema